src/privatesend/privatesend-server.h

```diff
@@ -25,6 +25,9 @@ class CPrivateSendServer : public CPrivateSendBaseSession, public CPrivateSendBa
     // Maximum number of participants in a certain session, random between min and max.
     int nSessionMaxParticipants;
 
+    //The sessions start time, used for variable participant count
+    int nSessionStartTime;
```

Comment:
Should drop `nSessionMaxParticipants` and replace it with `nSessionStartTime` in ctor below.